brickstrap.sh

```diff
@@ -214,9 +238,10 @@ function brp_sanity_check_cli_options()
 
 function brp_validate_cli_options()
 {
-    brp_sanity_check_cli_options "$@" && \
-    brp_validate_project_name && \
+    brp_sanity_check_cli_options "$@"
+    brp_validate_project_name
     brp_validate_component_names
+    brp_validate_qemu || [ $? -eq 255 ] # no QEMU specified = 255
```

Comment:
No can do. For `update-binfmts` to work you have to have a target binary. For you to have a target binary requires you to have performed the multistrap first.

The option assignment was broken, is now fixed, so validation works now. If you explicitly pass a `-Q` option or set `QEMU_STATIC` a misconfiguration is caught early (as intended).